npm/webpack-dev-server/tsconfig.json

```diff
@@ -45,9 +45,9 @@
     // "experimentalDecorators": true,        /* Enables experimental support for ES7 decorators. */
     // "emitDecoratorMetadata": true,         /* Enables experimental support for emitting type metadata for decorators. */
     "esModuleInterop": true,
+    "skipLibCheck": true,
     /** Allows us to strip internal types sourced from webpack */
-    "stripInternal": true,
-    "importsNotUsedAsValues": "error"
```

Comment:
Why was this removed?

Reply:
I was getting deprecation warnings and I think I forgot to migrate it to the new flag https://www.typescriptlang.org/tsconfig#preserveValueImports. Ill try adding it back in and see whats up